template-processors/base/bin/gitClone.sh

```diff
@@ -21,15 +21,15 @@ function pullFromTemplatesRepo {
   set +u
   if [ ! -z "$TEMPLATE_GIT_HTTP_PROXY" ] 
   then
-    http_proxy=$TEMPLATE_GIT_HTTP_PROXY
+    local http_proxy=$TEMPLATE_GIT_HTTP_PROXY
```

Comment:
Cool! Just, why the `local` and the subshell below, instead of just doing `export` here?

Reply:
argh, I see explanation in the commit message, sorry! :)